docs/developers-clear-containers-install.md

```diff
@@ -8,8 +8,7 @@ process.
 ## Requirements
 
   * [go 1.8.3](https://golang.org/)
-  * [glibc-static](https://www.gnu.org/software/libc/libc.html)
-  * [gcc](https://gcc.gnu.org/)
+  * [gcc](https://gcc.gnu.org/) (and associated C language build tooling such as `make`, `autoconf` and `libtool` which are required to build `cc-shim`).
```

Comment:
I recommend we remove the parentheses around the sentence following the link. If you think this looks strange, have ``go 1.8.3``  as the second bullet rather than the first.
Also, please limit line length to 78 characters (after "such").